### DISCUSSION TITLE
AX3600的固件不更了吗，好久没有了

### DISCUSSION THREAD

**Discussion by fujia521** [Boos4721/openwrt#167] [Q&A/答疑解惑]
AX3600的固件不更了吗，好久没有了

**Comment by Boos4721** [comment]
a litte busy, coming soon

**Comment by fujia521** [comment]
我很期待😊

发自我的iPhone

> 在 2022年7月31日，21:11，Boos4721 ***@***.***> 写道：
> 
> a litte busy, coming soon

**Comment by saocdr** [comment]
期待大佬更新
